python/requirements/ml/data-test-requirements.txt

```diff
@@ -14,3 +14,5 @@ google-api-core==1.34.0
 webdataset
 raydp==1.7.0b20231020.dev0
 pylance
+--extra-index-url https://test.pypi.org/simple
+hudi==0.1.0
```

Comment:
this is temporary for this PR to pass. the `--extra-index-url` will be removed once the official release is out